src/extension/options-page/Home.tsx

```diff
@@ -98,8 +98,6 @@ export default function DashboardHomePage() {
     )
 
     React.useEffect(() => {
-        if (personas.length !== 0) return
```

Comment:
I have no idea about why...

Reply:
An unconnected persona should goto **1A[6] "Connect Profile"** when restoring the backup.

Reply:
In fact 0 means `the data is not ready` so the following check logic will always execute at least once. Removing this line won't ensure the check logic more (but make the query happens every time the router change).

Reply:
Actually, we need to check if the persona is connected when every time the router changes.